go.mod

```diff
@@ -51,6 +51,8 @@ require (
 	gotest.tools/v3 v3.5.1
 )
 
+require github.com/dustinkirkland/golang-petname v0.0.0-20231002161417-6a283f1aaaf2
+
```

Comment:
why is this in its own section? no idea, but i ran `go mod tidy` and it didn't change, so 🤷

Reply:
We need to contribute a cannoicaliser for go.mod